.gitignore

```diff
@@ -18,31 +19,33 @@
 /storage/*
 !/storage/.keep
 
+# Ignore assets.
 /public/assets
-.byebug_history
 
-# RSpec testing
-/spec/.examples.txt
+# Ignore configuration files
+/config/database.yml
+/config/smtp.yml
 
-# Ignore environment and deployment files
-/.env*
-/.kamal/*
-/config/deploy.yml
+# Ignore database files
+/db/*.sqlite3
+/db/*.sqlite3-*
```

Comment:
the new default for these is /storage/, if we're changing Attachments.pwd (and others), should we change this too?